app/mailers/submission_mailer.rb

```diff
@@ -0,0 +1,37 @@
+require "#{Rails.root}/app/helpers/application_helper.rb"
+include ApplicationHelper
+
+class SubmissionMailer < ApplicationMailer
+
+  def new_submission_confirmation_email(submission)
+    @submission = submission
+
+    @form_name = @submission.form_name
+    if @form_name.downcase.include?("ask")
+      @form_contact = Organization.current_organization.ask_form_contact
+    elsif @form_name.downcase.include?("offer")
+      @form_contact = Organization.current_organization.offer_form_contact
+    elsif @form_name.downcase.include?("community_resource")
+      @form_contact = Organization.current_organization.community_resources_contact
+    end
+
+    @person = @submission.person
+    @locale = @person.preferred_locale || 'en'
+
+    @system_settings = SystemSetting.current_settings
```

Comment:
This is a class method that uses the first SystemSetting record as a User-admin-able way to change settings. seeds.rb includes creating one default record. This should be revisited to make sure we're all ok w this implementation.